react/modalcolumns.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import { ReactComponent as EnvOk } from './assets/envelope_ok.svg';
+import { ReactComponent as EnvBugged } from './assets/envelope_bugged.svg';
+
+class ModalColumns extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        }
+        this.chunkEnvelopesByBatchSize = this.chunkEnvelopesByBatchSize.bind(this);
+    }
+
+    chunkEnvelopesByBatchSize(batchSize, envelopes) {
+        return envelopes.reduce((acc, curValue) => {
+            if (acc.length !== 0 && acc[acc.length - 1].length < batchSize) {
+                return [
+                    ...acc.slice(0, -1),
+                    [
+                        ...acc[acc.length - 1],
+                        curValue
+                    ]
+                ];
+            }
+
+            return [
+                ...acc,
+                [curValue]
+            ]
+        }, []);
+    }
+
+    render() {
+        const resultTeam = this.chunkEnvelopesByBatchSize(this.props.batchSize, this.props.teamEnvelopes);
```

Comment:
nitpick: if you destructure variables from `this.props` before this line, you can use those shorthand variables here instead of typing out the longhand `this.props.batchSize`